zoleotracker/databaseSQL.py

```diff
@@ -0,0 +1,57 @@
+import mysql.connector
+from mysql.connector import Error
+
+def create_db_connection(host_name, user_name, user_password, db_name):
+    connection = None
+    try:
+        connection = mysql.connector.connect(
+            host=host_name,
+            user=user_name,
+            passwd=user_password,
+            database=db_name
+        )
+        print("MySQL Database connection successful")
+    except Error as err:
+        print(f"Error: '{err}'")
+
+    return connection
+
+def check_for_table(connection, table_name):
+    cursor = connection.cursor()
+    try:
+        cursor.execute("SHOW TABLES LIKE '%s' ", table_name)
+        result = cursor.fetchall()
+        if table_name in result:
+            return True
+        return False
+    except Error as err:
```

Comment:
where is the return in the case of error? if you except it, then you need to properly handle it, you will get back `None` from this. Do you really want to handle it in this level?